### ISSUE TITLE
Sprinting with PPB doc?

### ISSUE THREAD

**Issue by AstraLuma** [ppb/pursuedpybear#626]
Should we have a document talking about sprinting with PPB?

What should go in to it?

**Comment by ProsperousHeart** [comment]
Is this related to [this](https://ppb.dev/contributing.html)?

**Comment by AstraLuma** [comment]
Yes and no. Above and beyond that, the thought would be talking about how Piper and I run sprints, and cover things like You can just make a game!

**Comment by ProsperousHeart** [comment]
Would this be written in Sphynx then? At least to begin outlining the needs?

**Comment by AstraLuma** [comment]
Sphinx or website.
